dawe_vocabs/settings.py

```diff
@@ -92,311 +101,429 @@
+        "http://vocabs.paratoo.tern.org.au:1337/api/lut-condition-vegetation-diameter-classes",
         "Condition vegetation diameter classes codes",
         "A collection of condition vegetation diameter classes and its codes.",
         "fe0b8990-dc4c-4fc7-85e8-be08da5721a0",
         "Condition vegetation diameter",
     ),
     LUTSchema(
         "https://dev.core-api.paratoo.tern.org.au/lut-condition-vertebrate-pest-types",
+        "http://vocabs.paratoo.tern.org.au:1337/api/lut-condition-vertebrate-pest-types",
         "Condition vertebrate pest types codes",
         "A collection of condition vertebrate pest types and its codes.",
         "1eaaabf9-57a9-4c95-9cf3-58c20263b2a0",
         "Condition vertebrate pest",
+        "579449ad-4cea-4272-afa3-67f207941fb1",
+        "Condition vertebrate pest types",
+    ),
+    LUTSchema(
+        "http://vocabs.paratoo.tern.org.au:1337/api/lut-condition-vegetation-healths",
+        "Condition vegetation healths codes",
+        "A collection of condition vegetation healths and its codes.",
+        "b8379ff2-60dc-44cc-bf44-5ce7ed4bdd2f",
+        "Condition vegetation healths",
+    ),
+    LUTSchema(
+        "http://vocabs.paratoo.tern.org.au:1337/api/lut-condition-vertebrate-pest-presence-evidences",
+        "Condition vertebrate pest presence evidences codes",
+        "A collection of condition vertebrate pest presence evidences and its codes.",
+        "69638c57-1c38-47e1-8bae-c821411c3a30",
+        "Condition veterbrate pest presence evidences",
     ),
     LUTSchema(
         "https://dev.core-api.paratoo.tern.org.au/lut-fire-plot-burned-statuses",
+        "http://vocabs.paratoo.tern.org.au:1337/api/lut-fire-plot-burned-statuses",
         "Fire plot burned statuses codes",
         "A collection of fire plot burned statuses and its codes.",
         "5662a7dd-c1da-4659-8290-a1e6e42c879f",
         "Fire plot burned statuses",
     ),
     LUTSchema(
         "https://dev.core-api.paratoo.tern.org.au/lut-fire-substrate-types",
+        "http://vocabs.paratoo.tern.org.au:1337/api/lut-soils-substrates",
+        "Substrate types codes",
+        "A collection of substrate types and its codes.",
+        "0b89087f-345a-4e8d-8f84-38d5af499f10",
+        "Substrate",
+    ),
+    LUTSchema(
+        "http://vocabs.paratoo.tern.org.au:1337/api/lut-fire-substrate-types",
         "Fire substrate types codes",
         "A collection of fire substrate types and its codes.",
         "0b89087f-345a-4e8d-8f84-38d5af499f10",
+        "c7a0692c-113c-4593-ae04-5e49aba70fdf",
         "Fire substrate",
     ),
     LUTSchema(
         "https://dev.core-api.paratoo.tern.org.au/lut-fire-growth-forms",
+        "http://vocabs.paratoo.tern.org.au:1337/api/lut-fire-growth-forms",
         "Fire growth forms codes",
         "A collection of fire growth forms and its codes.",
         "032ed6fa-4c55-4752-b625-3e6b32672444",
         "Fire growth forms",
     ),
+    # LUTSchema(
```

Comment:
@junrongYu can you comment on why this is commented out?

Reply:
hi Edmond, because the api and the uuid are not used anymore

Reply:
If it's not used, can we just remove it entirely instead of having it commented out?

Reply:
yeah we can, can you delete it please?